steps/compliance/dwp-issuing-office/DWPIssuingOffice.js

```diff
@@ -60,6 +64,21 @@ class DWPIssuingOffice extends SaveToDraftStore {
         'Barrow IIDB Centre',
         'Barnsley Benefit Centre'
       ]);
+    } else if (getBenefitCode(this.journey.req.session.BenefitType.benefitType) === 'industrialDeathBenefit') {
+      return DWPIssuingOffice.selectify([
+        'Barrow IIDB Centre',
+        'Barnsley Benefit Centre'
+      ]);
+    } else if (getBenefitCode(this.journey.req.session.BenefitType.benefitType) === 'pensionCredits') {
+      return DWPIssuingOffice.selectify([
+        'Pensions Dispute Resolution Team',
+        'Recovery from Estates'
+      ]);
+    } else if (getBenefitCode(this.journey.req.session.BenefitType.benefitType) === 'retirementPension') {
+      return DWPIssuingOffice.selectify([
+        'Pensions Dispute Resolution Team',
+        'Recovery from Estates'
```

Comment:
Both pentionCredits and retirementRension returns same value. Can be joined with OR condition

Reply:
Updated